bindings/src/query_resp.rs

```diff
@@ -58,6 +58,24 @@ pub struct AmmSwapEstimationByDenomResponse {
     pub available_liquidity: Coin,
 }
 
+#[cw_serde]
+pub struct MarginOpenEstimationRawResponse {
+    pub position: i32,
+    pub leverage: String,
+    pub trading_asset: String,
+    pub collateral: Coin,
+    pub min_collateral: Coin,
+    // pub valid_collateral: bool, not found in the response from the chain
+    pub position_size: Coin,
+    pub swap_fee: String,
+    pub discount: String,
```

Comment:
I mean here, it is declared as string.

Reply:
so the query will get this field as string and then i convert them into decimal if the conversion fail because the value get is negative then we set the value at zero